boards/grand_central_m4/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - update path of Cargo config
+- bumped usb-device version to 0.3
```

Comment:
I would also definitely add that the HAL dependency has been upgraded to 0.17

Reply:
Whoops, looks like you committed as I was writing my comment.

Reply:
yeah realized that when I was looking things over, oops